src/logger/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+set(tgt acquire-zarr-logger)
```

Comment:
A couple of nits and FYIs:

- I believe this target name can be generalized to `acquire-logger`. It doesn't need to be Zarr-specific.
- Directly using the target name instead of a variable improves readability, maintainability, and the overall robustness of your CMake configuration. The indirection of a variable is unnecessary in most use cases and can introduce subtle bugs or confusion if you have multiple targets. Most of the time people use it to reduce redundancy, but it's typically unnecessary.